src/frontends/tensorflow/src/utils.cpp

```diff
@@ -34,13 +34,19 @@ void ov::frontend::tensorflow::set_out_name(const std::string& out_name, const o
 
 ov::op::PadType ov::frontend::tensorflow::convert_tf_padding(const ov::frontend::tensorflow::NodeContext& node,
                                                              const std::string& tf_padding) {
+    std::vector<std::string> supported_ops = {"Conv2D",
```

Comment:
Why vector and not a set? If set is used finding will be more straight forward: instead of `std::find` you can use `set::count`.

Reply:
Done